### ISSUE TITLE
Compatibility with Elixir 1.5.0

### ISSUE THREAD

**Issue by grych** [grych/drab#31]
Under 1.5.0, when using nested renders:

```
<%= render("users.html", users: @users, count: @count, color: @color) %>

<%= for user <- @users do %>
  <span @style.backgroundColor=<%=@color%>>
    User of <%= @count %>: <%= render("user.html", user: user) %><br>
  </span>
<% end %>
```

the EEx engine generates too many opening partials (`<span drab-partial=>`) without closing them.

Please do not use 1.5.0 until it is resolved.

**Comment by grych** [comment]
Reported it as an issue in 1.5.0: https://github.com/elixir-lang/elixir/issues/6391

**Comment by grych** [comment]
Fixed in elixir-lang/elixir#6391
